src/BaseCurve.sol

```diff
@@ -79,7 +92,7 @@ abstract contract BaseCurve {
     /// @param totalShares Total quantity of shares already awarded by the curve
     /// @return shares The number of shares equivalent to the given assets
     function convertToShares(uint256 assets, uint256 totalAssets, uint256 totalShares)
-        public
```

Comment:
3.20 - Functions in Curve Contracts Could Be external Instead of public

Made this `external`